packages/aws-cdk-lib/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -434,6 +443,7 @@ export class HttpApi extends HttpApiBase {
       corsConfiguration,
       description: props?.description,
       disableExecuteApiEndpoint: this.disableExecuteApiEndpoint,
+      routeSelectionExpression: props?.routeSelectionExpression === true ? '$request.method $request.path' : undefined,
```

Comment:
Why `$request.method $request.path` and not `${request.method} ${request.path}`?

It is probably correct as the deployment has not failed. However, the CFn documentation also says it in brackets, and the selection expression guide says the following.

> To use multiple variables in a selection expression, enclose the variable in brackets. For example, ${request.path.name} ${request.path.id}.

https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-parameter-mapping.html

Reply:
This is my mistakes! Thank you for your confirmation!!